components/charts/doughnut.js

```diff
@@ -56,12 +57,19 @@ export default function DoughnutChart(props) {
           options={{
             responsive: true,
             maintainAspectRatio: true,
-            legend: { display: false },
+            legend: { 
+              position:"right",
+              labels: {
+                usePointStyle: true,
```

Comment:
Can the legend be made of rounded squares? I think that could fit in really well

Reply:
aight sure ill look into that